### PR TITLE
Picker

### PR DESCRIPTION
**Pull request by cormacmccarthy**

### CODE REVIEW

File: js/placard.js [hunk 1]
```diff
@@ -242,7 +242,7 @@
 
 			this.$field.val(val);
 
-			if (!suppressEllipsis && !_isShown) {
+			if (!suppressEllipsis && !_isShown(this)) {
```
Comment (futuremint) [inline js/placard.js]: Is this change relevant to the Picker?

File: js/picker.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+				_showBelow(picker);
+			}
+		}
+
+		picker.$popup.css('visibility', 'visible');
+	};
+
+	var _showAbove = function _showAbove(picker) {
+		picker.$popup.css('height', (picker.options.height)?picker.options.height : DEFAULT_HEIGHT + 'px');
+		picker.$body.css('height', ((picker.options.height)?picker.options.height : DEFAULT_HEIGHT) - 73 + 'px');
```
Comment (futuremint) [inline js/picker.js]: What is "73" here? The answer to the life, the universe, and everything?

File: js/picker.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+		// OR use browser globals if AMD is not present
+		factory(jQuery);
+	}
+}(function ($) {
+	// -- END UMD WRAPPER PREFACE --
+
+	// -- BEGIN MODULE CODE HERE --
+
+	var old = $.fn.picker;
+	var EVENT_CALLBACK_MAP = { 'accepted': 'onAccept', 'cancelled': 'onCancel' };
```
Comment (futuremint) [inline js/picker.js]: This should have a descriptive, concise, single line comment about "why" this is here (as opposed to, say, just inlining the object?).

File: test/picker-test.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+		ok($picker.picker() === $picker, 'picker should be initialized');
+	});
+
+	test('should behave as expected - input', function(){
+		var $picker = $(html).find('#picker1');
+
+		$('body').append($picker);
+		$picker.picker();
+		$picker.on('cancelled.fu.picker', function(e, helpers){
+			ok(1===1, 'default action event (cancel) triggered upon external click');
```
Comment (futuremint) [inline test/picker-test.js]: How will you know when this assertion doesn't get called?

File: js/placard.js (hunk 1 above)
Comment (cormacmccarthy) [inline js/placard.js]: Not at this point, no.
